modelling/src/main/java/org/axonframework/modelling/saga/SagaMethodMessageHandlingMember.java

```diff
@@ -46,10 +46,16 @@
      * @param associationPropertyName The association property name to look up in the message
      * @param associationResolver     The association resolver configured for this handler
      * @param endingHandler           Flag to indicate if an invocation of the given handler should end the saga
+     * @deprecated in favour of the constructor not providing the {@code endingHandler} parameter, since that parameter
```

Comment:
If you are concerned about backwards compatibility on this one, then simply ignoring the value here isn't an option. It will change the behavior to a degree that you'd much rather have a compilation error.
Since this is not likely to be a component others use directly, I suggest removing the constructor instead.